examples/sdk/react/README.md

```diff
@@ -0,0 +1,64 @@
+# Getting Started with Create React App
```

Comment:
This shouldn't be here. Can we please adjust readme to our example?

Reply:
Fixed